client/src/main.rs

```diff
@@ -70,361 +63,23 @@ pub fn install_hooks() -> color_eyre::Result<()> {
 
     // used color_eyre's PanicHook as the standard panic hook
     let panic_hook = panic_hook.into_panic_hook();
-    panic::set_hook(Box::new(#[coverage(off)] move |panic_info| {
-        term::restore().unwrap();
-        panic_hook(panic_info);
-    }));
+    panic::set_hook(Box::new(
+        #[coverage(off)]
```

Comment:
why is this coverage off?

Reply:
`#[coverage(off)]` is not recursive, the top one doesn't coverage closures so you need to annotate every closure.